app/views/valuation/budget_investments/edit.html.erb

```diff
@@ -79,7 +79,7 @@
   <div class="row">
     <div class="small-12 medium-8 column">
       <%= f.label :valuation_finished do %>
-        <%= f.check_box :valuation_finished, title: t('valuation.budget_investments.edit.valuation_finished'), label: false %>
+        <%= f.check_box :valuation_finished, title: t('valuation.budget_investments.edit.valuation_finished'), label: false, id: 'js-investment-report-alert', "data-valuation-finished-alert": t("valuation.budget_investments.edit.valuation_finished_alert"), "data-valuation-finished-not-feasible-alert": t("valuation.budget_investments.edit.valuation_finished_not_feasible_alert") %>
```

Comment:
Remember remember the 100th of november 😬  (🤖  cries when lines are more than 100 chars long)